modules/services/utils/src/main/java/au/com/shiftyjelly/pocketcasts/utils/featureflag/Feature.kt

```diff
@@ -64,7 +64,7 @@ enum class Feature(
         title = "Deselect Chapters",
         defaultValue = BuildConfig.DEBUG,
         tier = FeatureTier.Plus(null),
-        hasFirebaseRemoteFlag = true,
+        hasFirebaseRemoteFlag = false,
```

Comment:
I also set Deselect Chapter remote feature flag to false so that the unfinished feature is not shown in v7.57.1 when remote feature flag is enabled in future.